deploy/template.yaml

```diff
@@ -0,0 +1,179 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  ipv-cri-uk-passport-back
+
+Globals:
+  Function:
+    Timeout: 20
+
+Parameters:
+  Environment:
+    Type: String
+
+Resources:
+
+  IPVCriUKPassportAPI:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: IPV CRI UK Passport API Gateway
+      StageName: !Sub ${Environment}
+
+  IPVDcsCredentialFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-dcs-credential-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.lambda.DcsCredentialHandler::handleRequest
```

Comment:
I think this handler pathname is incorrect.

Should it be uk.gov.di.ipv.cri.passport.dcscredential.DcsCredentialHandler::handleRequest